docs/development.md

```diff
@@ -25,15 +25,15 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
 Our JavaScript code is written in modular ECMAScript 6 (ES6) syntax, which is not supported in many of the existing Web browsers today.<br>
 To resolve this, we need to *bundle and transpile* ("build" afterwards) them into standard ECMAScript 5 which is supported by (almost) all browsers.
 
-Run the following command to build the JavaScript files for the application's use:
+Run the following command to build the JavaScript files for the application's use in development mode:
 ```sh
-npm run build
+npm start
```

Comment:
`npm start` is a shortform. I think only the `start` keyword allows this shortform. Should I stick to `npm run start` for consistency?

Reply:
Ah right, I totally forgot about that (what's more I'm currently developing Angular app 🤦‍♂️ ).

I won't really call it `start` though, since for Node.js developers it has a somewhat implicit meaning of starting a dev server. Until we use Webpack dev server, whenever that is, let's name it something else for now. Or does this PR also include Webpack dev server?

Reply:
Does `npm run dev` sound good? 

Webpack dev server will be included in the next PR, but this PR allows for different dev tools to be added more easily and also adds source-map.

As an example, by adding `watch:true`, we can allow `npm run dev` to be automatically run everytime file changes. I'll leave that as a possible workaround to be done by developers after this change, but I'm not including it as part of the PR as it will be temporary until webpack dev server is added in another issue

Reply:
`npm run dev` sounds good. Let's refrain from `start` until we explicitly support dev server.